app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :entities
+  has_many :groups
+  has_one_attached :avatar
+
+  validates :name, presence: { message: "Firstname can't be blank!" }, length: { minimum: 3, maximum: 250 }
+  validates :email, presence: true
+end
```

Comment:
- [ ] [OPTIONAL] Kindly enable `:confirmable`. Email confirmation is important because it helps to ensure that the email address provided by the user during registration is valid and belongs to them. It also helps to prevent spam or fake accounts from being created.
- By using the `:confirmable` module in Devise, web applications can ensure that only legitimate users can access the site's resources and functionalities, and can prevent unauthorized access or misuse of the site.

Let's fix/implement this 💪🏽